### PR TITLE
Various improvements to build and docs

### PR DESCRIPTION
**Pull request by cpsievert**

### CODE REVIEW

File: tools/yarn_install.R [hunk 1]
```diff
@@ -1,9 +1,129 @@
...
+
+add_prefixes <- function(src, property, ok_values = NULL) {
+  pattern <- paste0("^\\s*", property, ":\\s*(.+);")
+  idx <- grep(pattern, src)
+  for (i in idx) {
+    prop <- src[[i]]
+    if (length(ok_values)) {
+      value <- regmatches(prop, regexec(pattern, prop))[[1]][2]
+      vals <- strsplit(value, "\\s+")[[1]]
+      if (all(grepl(paste(ok_values, collapse = "|"), vals))) next
```
Comment (jcheng5) [inline tools/yarn_install.R]: 1. Allow `!important` as a value
2. Just use `%in%`, not `grepl`

File: tools/yarn_install.R [hunk 2]
```diff
@@ -1,9 +1,129 @@
...
+
+# ----------------------------------------------------------------------
+# Check to make sure we're not missing any vendor prefixes
+# that we don't already know about in the distributed CSS
+# ----------------------------------------------------------------------
+
+find_prefixed_css <- function(css) {
+  vendors <- c("webkit", "moz", "ms")
+  prefixes <- lapply(vendors, function(vendor) {
+    pattern <- sprintf("-%s-([a-z-]+)", vendor)
```
Comment (jcheng5) [inline tools/yarn_install.R]:
```suggestion
    pattern <- sprintf("-%s-([^:]+)", vendor)
```

Comment (cpsievert) [reply tools/yarn_install.R]: There are a few cases where this won't work 

https://github.com/twbs/bootstrap/blob/af4e37631/dist/css/bootstrap.css#L1632
https://github.com/twbs/bootstrap/blob/af4e37631c526061d853881f11c5363f61907ba5/dist/css/bootstrap.css#L5575
https://github.com/twbs/bootstrap/blob/af4e37631/dist/css/bootstrap.css#L5767

Comment (cpsievert) [reply tools/yarn_install.R]: How about we compromise on the following?

```r
pattern <- sprintf("-%s-([^:|;| |\\)]+)", vendor)
```

**Pull request merged**